go.mod

```diff
@@ -3,36 +3,49 @@ module github.com/ipni/dhstore
 go 1.19
 
 require (
+	contrib.go.opencensus.io/exporter/prometheus v0.4.2
```

Comment:
We should use opentelemetry.io going forward. The other flavours are deprecated. 

Look at an example here: https://github.com/ipni/heyfil/blob/main/metrics.go